inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/sidebar/RecommendationSidebar.java

```diff
@@ -67,5 +91,98 @@ public RecommendationSidebar(String aId, IModel<AnnotatorState> aModel,
                 aAnnotationPage.actionRefreshDocument(_target)));
         
         add(form);
-    }
+
+		chartContainer = new Label("chart-container" );
+		add(chartContainer);
+
+		add(new AbstractAjaxTimerBehavior(Duration.milliseconds(5000)) {
+			private static final long serialVersionUID = -3782208159226605584L;
+
+			@Override
+			protected void onTimer(AjaxRequestTarget target) {
+
+				List<LoggedEvent> loggedEvents = recommendationService.listLoggedEvents(aModel.getObject().getProject(),
+						aModel.getObject().getUser());
+
+				// iterate over the logged events to extract the scores only.
+				List<Double> listDetailJson = new ArrayList<Double>();
+
+				for (LoggedEvent loggedEvent : loggedEvents) {
+					String detailJson = loggedEvent.getDetails();
+
+					ObjectMapper mapper = new ObjectMapper();
+					Double scoreDouble;
+					try {
+						LoggedEventDetail detail = mapper.readValue(detailJson, LoggedEventDetail.class);
+
+						try {
+							scoreDouble = Double.valueOf(detail.getScore());
+							listDetailJson.add(scoreDouble);
+						} catch (NumberFormatException e) {
+							log.debug("Skipping logged Event due to invalid score, Detail:{}", detail);
+							continue;
+						}
+
+					} catch (IOException e) {
+						log.error(e.toString(), e);
+					}
+				}
+
+				String data = "";
```

Comment:
Better use Jackson (i.e. `JSONUtil.toJson(...)`) to serialize the list to JSON.

Reply:
Done.